src/main/java/org/sefglobal/scholarx/config/SecurityConfig.java

```diff
@@ -47,6 +52,15 @@ protected void configure(HttpSecurity http) throws Exception {
       .permitAll()
       .and()
       .logout()
+            .logoutSuccessHandler(new LogoutSuccessHandler() {
+              @Override
+              public void onLogoutSuccess(HttpServletRequest request, HttpServletResponse httpServletResponse, Authentication authentication) throws IOException, ServletException {
```

Comment:
break this line to not increase the character limit of a line

Reply:
Sure, on it

Reply:
Resolved with the commit a074c37 Please Review